modules/operator-openshift/README.md

```diff
@@ -76,7 +98,7 @@ locals {
 }
 
 module "crowdstrike_operator" {
-  source = "github.com/CrowdStrike/terraform-kubectl-falcon//modules/operator?ref=v0.4.0"
+  source = "github.com/CrowdStrike/terraform-kubectl-falcon//modules/operator?ref=v0.1.0"
```

Comment:
Did we mean to go from version `v0.4.0` back to version `v0.1.0`?

Reply:
Nope, I assume this was modified by terraform docs